docs/introduction/quick-start.md

```diff
@@ -27,6 +27,16 @@ yarn add react-redux
 
 You'll also need to [install Redux](https://redux.js.org/introduction/installation) and [set up a Redux store](https://redux.js.org/recipes/configuring-your-store/) in your app.
 
+## Create an App
```

Comment:
```suggestion
### Using Create React App
```

Let's make this an h3, so it's part of the overall installation section. Otherwise it looks like you do both this and the install command before it.

We should also move it above the vanilla install steps, so it's the first thing and fits in as the "recommended" option.

Reply:
I've updated the docs for [Redux](https://redux-toolkit.js.org/introduction/quick-start#create-a-react-redux-app) and [Redux Toolkit](https://redux.js.org/introduction/getting-started#redux-toolkit), and they still have the template recommendation after installation with the same heading level. Should I update those pages to be consistent?